cmd/root.go

```diff
@@ -120,6 +120,7 @@ Common Management Commands:
 	rootCmd.AddCommand(newReposCmd())
 	rootCmd.AddCommand(newSearchCmd())
 	rootCmd.AddCommand(newStatusCmd())
+	rootCmd.AddCommand(newHealthCmd())
```

Comment:
And this should not be merged with status somehow?

Reply:
Status is currently and open endpoint, while health is closed. They are somewhat different enough to be separate commands?

Reply:
Makes sense. I just think from a UX point of view we should consider grouping them